aws/api/ecs.tf

```diff
@@ -0,0 +1,97 @@
+locals {
+  container_env     = [] # TODO: app api environment variables
+  container_secrets = [] # TODO: app api secrets
+}
+
+module "api_ecs" {
+  source = "github.com/cds-snc/terraform-modules//ecs?ref=50c0f631d2c8558e6eec44138ffc2e963a1dfa9a" # v9.6.0
+
+  create_cluster = false
+  cluster_name   = var.ecs_cluster_name
+  service_name   = "forms-api"
+  task_cpu       = 1024
+  task_memory    = 2048
+
+  # Scaling
+  enable_autoscaling       = true
+  desired_count            = 1
+  autoscaling_min_capacity = 1
+  autoscaling_max_capacity = 3
+
+  # Task definition
+  container_image       = "${var.api_image_ecr_url}:${var.api_image_tag}"
+  container_host_port   = 3000
+  container_port        = 3000
+  container_environment = local.container_env
+  container_secrets     = local.container_secrets
+
+  task_exec_role_policy_documents = [
+    data.aws_iam_policy_document.api_ecs_dynamodb_vault.json,
+    data.aws_iam_policy_document.api_ecs_kms_vault.json,
+    data.aws_iam_policy_document.api_ecs_s3_vault.json,
+  ]
+
+  # Networking
+  lb_target_group_arn = var.lb_target_group_arn_api_ecs
+  subnet_ids          = var.private_subnet_ids
```

Comment:
wil that allow us to hit the idp to validate the token?

Reply:
Yup, when I setup the security group for the API, it will be able to make outbound connections to the internet on 443 using the subnet's NAT gateway.